server/configuration.go

```
@@ -270,7 +270,10 @@ func (p *Plugin) ensureDemoUser(configuration *configuration) (string, error) {
 
 	for _, team := range teams {
 		// Ignore any error.
-		p.API.CreateTeamMember(team.Id, configuration.demoUserId)
+		_, err := p.API.CreateTeamMember(team.Id, configuration.demoUserID)
+		if err != nil {
+			p.API.LogError("Failed add demo user to team", "teamID", team.Id, "error", err.Error())
+		}
```

Shouldn't we remove the `//Ignore any error` comment?